docs/quickstart_zh.md

```diff
@@ -60,7 +60,9 @@ dtm init -f config.yaml
 2022-03-04 12:08:20 ✔ [SUCCESS]  Initialize finished.
 ```
 
-这一步会根据配置文件来下载需要的插件。
+- 这一步会根据配置文件来下载需要的插件。
+- 如果dtm和远端的dtm不一致，将不会下载插件。
```

Comment:
Here it's not very accurate. Translate the English version from the previous comment.

Reply:
Updated